omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,111 @@
+Administrators with restricted permissions
+==========================================
+
+
+Summary
+-------
+
+An administrator with restricted permissions is a way to cater for the need for more powerful users. In the real world, these will typically be imaging facility managers, image analysts, or anybody who needs to organize users and data in OMERO. The only other way how to create users with necessary powers for abovementioned actions would be to create new administrators in OMERO, which is not advisable for security reasons. Administrators with restricted permissions can be created and and their permissions tailored for the task they are meant to perform on behalf of other OMERO users.
+
+Typically, a full administrator in OMERO would create new administrators with restricted permissions using OMERO.web interface (hook to Help with Webadmin graphic screenshots)
```

Comment:
Suggest: "Full administrators in OMERO can create new administrators with restricted permissions using OMERO.web interface".